Forge/src/main/java/me/ferdz/placeableitems/network/SUpdateFluidHolderPacket.java

```diff
@@ -0,0 +1,81 @@
+package me.ferdz.placeableitems.network;
+
+import me.ferdz.placeableitems.tileentity.FluidHolderTileEntity;
+
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.registries.ForgeRegistries;
+
+/**
+ * A server -> client packet to update the state of a {@link FluidHolderTileEntity}.
+ *
+ * @author Parker Hawke - Choco
+ */
+public class SUpdateFluidHolderPacket {
+
+    private BlockPos tilePos;
+    private FluidStack fluidStack;
+
+    /**
+     * Create a new SUpdateFluidHolderPacket with the tile's position and fluid stack.
+     *
+     * @param tilePos the tile's position
+     * @param fluidStack the fluid stack
+     */
+    public SUpdateFluidHolderPacket(BlockPos tilePos, FluidStack fluidStack) {
+        this.tilePos = tilePos;
+        this.fluidStack = fluidStack;
+    }
+
+    /**
+     * Create a new SUpdateFluidHolderPacket based on the values supplied by the tile entity.
+     *
+     * @param tile the tile entity whose values may be used to populate this packet data
+     */
+    public SUpdateFluidHolderPacket(FluidHolderTileEntity tile) {
```

Comment:
Would you mind refactoring this so it spells out the name completely? `tile` would become `tileEntity`